Description: Avoid pypi downloads during build
 Patches setup.py to remove things from requirements, because otherwise, the
 unit tests are downloading from pypi.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-03-10

--- python-pecan-0.4.5.orig/setup.py
+++ python-pecan-0.4.5/setup.py
@@ -43,22 +43,22 @@
 
 tests_require = requirements + [
     'virtualenv',
-    'gunicorn',
+#    'gunicorn',
     'mock',
     'sqlalchemy'
 ]
 if sys.version_info < (2, 7):
     tests_require += ['unittest2']
 
-if sys.version_info < (3, 0):
-    # These don't support Python3 yet - don't run their tests
-    if platform.python_implementation() != 'PyPy':
-        # Kajiki is not pypy-compatible
-        tests_require += ['Kajiki']
-    tests_require += ['Genshi']
-else:
-    # Genshi added Python3 support in 0.7
-    tests_require += ['Genshi>=0.7']
+#if sys.version_info < (3, 0):
+#    # These don't support Python3 yet - don't run their tests
+#    if platform.python_implementation() != 'PyPy':
+#        # Kajiki is not pypy-compatible
+#        tests_require += ['Kajiki']
+#    tests_require += ['Genshi']
+#else:
+#    # Genshi added Python3 support in 0.7
+#    tests_require += ['Genshi>=0.7']
 
 if sys.version_info < (3, 0) or sys.version_info >= (3, 3):
     tests_require += ['Jinja2']
